shared/libblight_protocol/libblight_protocol.cpp

```
@@ -0,0 +1,883 @@
+        std::shared_lock lock(ackQueuesMutex);
+        if(ackQueues.contains(fd)){
+            errno = EINVAL;
+            return nullptr;
+        }
+        auto stop = new std::atomic_bool{false};
+        blight_thread_t* thread;
+        {
+            std::mutex mutex;
+            std::unique_lock ulock(mutex);
+            std::condition_variable condition;
+            thread = new blight_thread_t{
```

Cppcheck (2.10-2)

[error] Uninitialized variable

(legacyUninitvar)